hs_core/models.py

```diff
@@ -2007,6 +2009,46 @@ def update(cls, element_id, **kwargs):
 
         super(FundingAgency, cls).update(element_id, **kwargs)
 
+    @sync_to_async
+    def get_funding_records(self, apps):
+        FundingAgency = apps.get_model('hs_core', 'FundingAgency')
+        return list(FundingAgency.objects.all())
+
+    @classmethod
+    async def update_record(self, record):
```

Comment:
also a nomenclature recommendation:
this is a classmethod, so instead of `self` we typically use `cls` parameter